lib/runbook/views/markdown.rb

```diff
@@ -23,7 +23,7 @@ def self.runbook__entities__step(object, output, metadata)
 
     def self.runbook__statements__ask(object, output, metadata)
       default_msg = object.default ?  " (default: #{object.default})" : ""
-      output << "   #{object.prompt} into #{object.into}#{default_msg}\n\n"
+      output << "   #{object.prompt} into `#{object.into}#{default_msg}`\n\n"
```

Comment:
Here I believe we wouldn't want `#{default_msg}` within the backticks.

Reply:
ack